File: hello-world/hello.jl.py
#!/usr/bin/env python3
#= Code visible only to Python
println = print
println("Hello from \x1b[36mPython\x1b[m!")

''' Code visible only to Julia =#
println("Hello from \x1b[35mJulia\x1b[m!")

# Code visible to Python and Julia '''
println("Hello from \x1b[36mPython\x1b[m and \x1b[35mJulia\x1b[m!")

File: hello-world/hello_jl.nim
#!/usr/bin/env nim
#= code visible to Nim

echo("Hello from \x1b[93mNim\x1b[m!")

#[ code visible to Julia =#

echo = println
echo("Hello from \x1b[35mJulia\x1b[m!")

# code visible to Julia and Nim ]#

echo("Hello from \x1b[35mJulia\x1b[m and \x1b[93mNim\x1b[m!")
